tests/framework/installer/ceph_installer.go

```diff
@@ -571,6 +571,9 @@ func (h *CephInstaller) UninstallRookFromMultipleNS(systemNamespace string, name
 
 		roles := h.Manifests.GetClusterRoles(namespace, systemNamespace)
 		_, err = h.k8shelper.KubectlWithStdin(roles, deleteFromStdinArgs...)
+		if err != nil {
+			logger.Errorf("failed to create rook external cluster. %v ", err)
```

Comment:
failed to create? Is that what the command is doing?

Reply:
oops, it deleting cluster roles. Thanks.